### DISCUSSION TITLE
"no:assignee" is unintuitive change to assignee: none

### DISCUSSION THREAD

**Discussion by askpatrickw** [community/community#5498] [Projects and Issues]
It appears as though the issues filter is of the form PROPERTY: VALUE.

![image](https://user-images.githubusercontent.com/4002194/132253937-b1129c09-dbd6-4c05-8430-5d4a4f171aa9.png)

assignee: none, or null, or "" would easier to use, discover and remember.

**Comment by bfontaine** [comment]
Note that @null is a valid username.

**Reply by askpatrickw** [reply]
I was just thinking.  Another approach would be to add no:property to the examples in the filter popover help...
